### ISSUE TITLE
add_debpackages skipping inclusion of some packages

### ISSUE THREAD

**Issue by dburm** [openSUSE/libsolv#132]
For trusty repositories:

Actual number on packages:
```
$ grep -c '^Package' Packages.trusty.{main,universe,multiverse,restricted} ; cat Packages.trusty.{main,universe,multiverse,restricted} | grep -c '^Package'
```
> Packages.trusty.main:8566
> Packages.trusty.universe:35524
> Packages.trusty.multiverse:741
> Packages.trusty.restricted:49
> 44880

After adding packages to the pool:
```
for repo in repos:
    print repo.name, repo.nsolvables
    n += repo.nsolvables
print n
```

> trusty.main 8565
> trusty.universe 35522
> trusty.multiverse 741
> trusty.restricted 49
> 44877

Missed packages are:
chef-zero, libvisual-0.4-0, monodoc-gmime2.6-manual

**Comment by dburm** [comment]
I've found workaround: need to split Packages index file to several lower sized files.

**Comment by mlschroe** [comment]
Ok, but it would be nice to fix the underlying bug as well ;)

**Comment by mlschroe** [comment]
I think I've fixed this with commit 30b06921106ba2202da36cb1d45e08c95b4daf43

Thanks for reporting this!

**Comment by dburm** [comment]
Confirmed, works good. Thanx a lot!
